package_maximizer/parsers/brew_parser.py

```
@@ -117,57 +114,59 @@ def _parse_brew_info(self, raw: str) -> list[Package]:
         """
         packages = []
         current_pkg = None
-        lines = raw.strip().split('\n')
-        
+        lines = raw.strip().split("\n")
+
         for line in lines:
             line = line.strip()
-            
+
             # Проверяем, начинается ли строка с имени пакета (с двоеточием)
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)